app/models/form.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: forms
+#
+#  id              :bigint           not null, primary key
+#  description     :text
+#  name            :string
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  organization_id :bigint
+#
+# Indexes
+#
+#  index_forms_on_organization_id  (organization_id)
+#
+class Form < ApplicationRecord
+  has_many :form_questions, dependent: :destroy
+  has_many :questions, through: :form_questions
+  has_many :submissions, dependent: :destroy
+
+  belongs_to :organization
```

Comment:
I think we want to use `acts_as_tenant` for all resources that belong to an organization so they are using the gem and scope appropriately. I believe then we don't have to use `current.organization...` and can just call Resource.first or whatever we need.

Reply:
Ah ok. I was under the impression that `acts_as_tenant` was only for when we use multiple domains or sub-domains. But we're not, right? I guess if it adds convenience with scoping then it makes sense to keep in. I'll update this line.

Reply:
yes I asked @edwinthinks about this and his recommendation was all belongs_to org associations should use acts_as_tenant